vllm/hpu/ops.py

```diff
@@ -126,6 +133,21 @@ def static_fused_moe(hidden_states, w1, w2, score, topk):
     return final_hidden_states.view(-1, D)
 
 
+#TODO: remove after SW-195415 fix
```

Comment:
I would suggest to remove this comment -- SW-195415 doesn't say anything. Please try to describe issue instead

Reply:
done